felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -767,6 +801,20 @@ func (m *bpfEndpointManager) updateWEPsInDataplane() {
 			defer wg.Done()
 			defer sem.Release(1)
 			err := m.applyPolicy(ifaceName)
+			if err != nil {
+				if isLinkNotFoundError(err) {
+					// Interface is gone, nothing to do.
+					log.WithField("ifaceName", ifaceName).Debug(
+						"Ignoring request to program interface that is not present.")
+					err = nil
+				}
+			} else {
+				m.ifacesLock.Lock()
+				if iface, ok := m.nameToIface[ifaceName]; !ok || iface.info.ifindex == 0 {
```

Comment:
I think this case will match with `applyPolicy`'s existing check on whether the interface is up according to `iface.info.ifaceIsUp` which is true exactly when the index is set.

Previously we handled that quietly because it's 100% expected that felix will see WEPs for pods that are being deleted in the dataplane (so their interfaces are missing) so need to be careful not to log lots of new errors here.